docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -410,12 +448,83 @@ A key/value list is a way of storing tagged item pairs within a list. For exampl
 
 The key values must be unique within a kvlist, as the data is stored as a JSON object with the key becoming the property name and the value of the property the corresponding value for the key.
 
+A returned list with 4 entries would have a value as shown below.
+
+.. code-block:: JSON
+
+   {
+        "idle" : "\"speed == 0\",
+        "operational" : \"speed == 5000\",
+        "transitional" : \"speed > 0 && speed < 5000\",
+        "overspeed" : \"speed > 5000\"
+   }
```

Comment:
Not a valid JSON, so a highlighting issue

Reply:
Also for type kvlist and having items string define in above example would return only entry
```
{"idle": "\"speed == 0\""}
```
Either update the example or its result

Reply:
The returned value is when the user has defined 4 entries in the kit, it is not the default.